Rubberduck.CodeAnalysis/Inspections/Concrete/MultipleDeclarationsInspection.cs

```diff
@@ -20,6 +20,11 @@ namespace Rubberduck.CodeAnalysis.Inspections.Concrete
     /// Dim foo As Long, bar As Long
     /// ]]>
     /// </module>
+    /// <module name="MyModule" type="Standard Module">
```

Comment:
That's a very good start, the intention was to have it as a separate `<example>` element. As such the change you performed is correct, but still missing the additional `<example>` tags (including a `hasResult="true"`) to separate it from the existing example that generates a result

Reply:
Okay! let me change it Thank  u!

Reply:
I take ArgumentWithIncompatibleObjectTypeInspection class an it was different . My fault ^^

Reply:
That's because that particular inspection involves multiple modules and so each example needs to have multiple modules associated with it.